src/displayapp/screens/WatchFacePineTimeStyle.cpp

```diff
@@ -431,16 +500,19 @@ void WatchFacePineTimeStyle::Refresh() {
   stepCount = motionController.NbSteps();
   motionSensorOk = motionController.IsSensorOk();
   if (stepCount.IsUpdated() || motionSensorOk.IsUpdated()) {
-    lv_gauge_set_value(stepGauge, 0, (stepCount.Get() / (settingsController.GetStepsGoal() / 100)));
+    lv_gauge_set_value(stepGauge, 0, (stepCount.Get() / (settingsController.GetStepsGoal() / 100)) % 100);
```

Comment:
I'd remove this modulo, keeping this PR to just the gauge settings

Reply:
The gauge modulo has been discussed previously in #872, I'm in favor of it for the reasons outlined there. If it is deeply unpopular I will add a setting toggle for it, but I feel that the gauge continuing to do _something_ once the step goal is reached is an improvement over the current version.

Reply:
thanks for the background info, resolved